### PR TITLE
[feat]: grid done

### PR DESCRIPTION
**Pull request by ranitasahaa1230**
Please review my grid component!

### CODE REVIEW

File: Components/Grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
+.grid-box {
+    border: 1px solid var(--gray);
+    margin: auto;
+  padding: 1.5rem;
+  border-radius: 6px;
+  margin-bottom: 1.5rem;
+  }
```
Comment (rohanmathur91) [inline Components/Grid/grid.css]: indentation is not proper

File: Components/Grid/grid.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+.grid-2-col{
+    display: grid;
+    grid-template-columns: repeat(2, minmax(0,1fr));
```
Comment (rohanmathur91) [inline Components/Grid/grid.css]: In minmax(0, 1fr), why you have put zero for min value, can you please explain?

File: Components/Grid/grid.css [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+}
+.card-vertical{
+    box-shadow: 0 2px 2px var(--online);
+}
+.card-content{
+text-align: center;
+}
+.card-grid{
+    font-size: 1.6rem;
+   margin: 4px;
```
Comment (rohanmathur91) [inline Components/Grid/grid.css]: indentation is not correct.

**Review by rohanmathur91 — CHANGES_REQUESTED** [review]
Added some comments.
Also documentation content is overlapping.

image

**Review by maitrakhatri — COMMENTED** [review]
take care of things that Rohan has mentioned, rest looks good

**Pull request merged**